src/test/developTest/java/mServer/crawler/sender/arte/ArteCategoryFilmListDeserializerTest.java

```diff
@@ -16,36 +16,35 @@
 @RunWith(Parameterized.class)
 public class ArteCategoryFilmListDeserializerTest {
 
-    @Parameterized.Parameters
-    public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][] {   
-            { "/arte/arte_films_of_category_page1.json", new String[] { "073658-000-A", "053299-141-A", "072401-061-A", "072401-059-A", "072401-060-A", "076512-000-A", "076511-000-A", "076510-000-A", "053299-140-A", "072401-058-A" }, 23 },
-            { "/arte/arte_films_of_category_page_last.json", new String[] { "072401-011-A" }, 3 },
-        });
-    }
-    
-    private final String jsonFile;
-    private final String[] expectedProgramIds;
-    private final int expectedPages;
-    private final ArteCategoryFilmListDeserializer target;
-    
-    public ArteCategoryFilmListDeserializerTest(String aJsonFile, String[] aProgramIds, int aPages) {
-        jsonFile = aJsonFile;
-        expectedProgramIds = aProgramIds;
-        expectedPages = aPages;
-        this.target = new ArteCategoryFilmListDeserializer();
-    }
-    
-    @Test
-    public void testDeserialize() {
-        
-        JsonObject jsonObject = JsonFileReader.readJson(jsonFile);
-        
-        ArteCategoryFilmsDTO actual = target.deserialize(jsonObject, ArteCategoryFilmsDTO.class, null);
-        
-        assertThat(actual, notNullValue());
-        assertThat(actual.getPages(), equalTo(expectedPages));
-        ArrayList<String> actualProgramIds = actual.getProgramIds();
-        assertThat(actualProgramIds, Matchers.containsInAnyOrder(expectedProgramIds));
-    }   
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][]{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)